internal/api/api.go

```diff
@@ -190,3 +224,19 @@ func (api *idler) getURLAndToken(r *http.Request) (string, string, error) {
 	}
 	return "", "", fmt.Errorf("Unknown or invalid OpenShift API URL")
 }
+
+func (api idler) isJenkinsUnIdled(openshiftURL, openshiftToken, namespace string) (bool, error) {
+	state, err := api.openShiftClient.IsIdle(openshiftURL, openshiftToken, namespace, "jenkins")
+	if err != nil {
+		return false, err
+	}
+
+	status := state == model.JenkinsStarting || state == model.JenkinsRunning
```

Comment:
@sthaha Eventhough it looks good to me, let's be sure about the precedence here https://golang.org/ref/spec#Operator_precedence

Is go-format getting rid of brackets here?

Reply:
@kishansagathiya I didn't put brackets  because the precedence is clear to me and tbh we don't need to make any simpler ... If there is guideline I am happy to follow it but that is beyond the scope of this change.

Reply:
cool then